.github/workflows/ci.yml

```diff
@@ -51,6 +51,14 @@ jobs:
           make clean && make doctest
         working-directory: docs/
 
+      - name: Ensure docs build - PyVortex
+        run: |
+          set -ex
+
+          (cd pyvortex && uv run maturin develop)
```

Comment:
we should have already built it?

Reply:
I think so? `uv run pytest` should have the side effect of building and installing it, which is what develop does.